packages/SidePanel/src/components/Header/Header.js

```diff
@@ -23,57 +23,49 @@ const defaultProps = {
   level: 2,
   isCompact: false,
   onClose: () => {},
+  refHeading: null,
 };
 
-function darkBackgroundProps(kind) {
-  if (kind === Button.Kinds.PRIMARY) {
-    return { isDark: true };
-  }
-
-  return {};
-}
-
 const Header = React.forwardRef((props, ref) => {
   const {
+    children,
     hasCloseButton,
     getPushContentElement,
     isCompact,
     kind,
     level,
-    // injected by Dialog.js
     onClose,
+    refHeading,
     ...moreProps
   } = props;
 
   return (
     <sc.Header
       data-pka-anchor="sidepanel.header"
       hasPushedElement={!!getPushContentElement}
-      ref={ref}
-      kind={kind}
       isCompact={isCompact}
+      kind={kind}
+      ref={ref}
       {...moreProps}
     >
-      <Heading tabIndex="-1" level={level} displayLevel={isCompact ? 4 : 3}>
-        {props.children}
+      <Heading level={level} displayLevel={isCompact ? 4 : 3} ref={refHeading}>
+        {children}
       </Heading>
-      <div>
-        {hasCloseButton ? (
-          <Button.Close
-            data-pka-anchor="sidepanel-header-close"
-            isSemantic={false}
-            onClick={onClose}
-            {...darkBackgroundProps(kind)}
-            size={isCompact ? ShirtSizes.SMALL : ShirtSizes.MEDIUM}
-          />
-        ) : null}
-      </div>
+      {hasCloseButton && (
+        <Button.Close
+          data-pka-anchor="sidepanel.header.close"
```

Comment:
This was updated from `sidepanel-header-close` to align with our naming conventions.